Sources/Recycling/GridViews/Ads/Cell/AdsGridViewCell.swift

```diff
@@ -147,7 +162,11 @@ public class AdsGridViewCell: UICollectionViewCell {
             titleLabel.leadingAnchor.constraint(equalTo: leadingAnchor),
             titleLabel.trailingAnchor.constraint(equalTo: trailingAnchor),
             titleLabel.heightAnchor.constraint(equalToConstant: AdsGridViewCell.titleHeight),
-            titleLabel.bottomAnchor.constraint(equalTo: bottomAnchor, constant: -AdsGridViewCell.titleBottomMargin),
+
+            accessoryLabel.topAnchor.constraint(equalTo: titleLabel.bottomAnchor),
+            accessoryLabel.leadingAnchor.constraint(equalTo: leadingAnchor),
+            accessoryLabel.trailingAnchor.constraint(equalTo: trailingAnchor),
+            accessoryLabel.bottomAnchor.constraint(equalTo: bottomAnchor, constant: -AdsGridViewCell.bottomMargin),
```

Comment:
Should the `accessoryLabel` also have its `heightAnchor` constrained? I see the others have it.

Reply:
No, because this label is optional in contrast with others, so I want the height to be zero when there is no text set.

Reply:
Ah, of course! Don't mind me😎